meta/src/model/meta_admin.rs

```diff
@@ -414,11 +414,11 @@ impl AdminMeta {
             id: self.config.node_basic.node_id,
             grpc_addr: build_address(
                 self.config.host.clone(),
-                self.config.cluster.grpc_listen_port,
+                self.config.cluster.grpc_listen_port.unwrap(),
```

Comment:
Config `grpc_listen_port` may be none but here trying to give `grpc_addr` an exact value.